src/components/AgreementLineSections/Coverage/Coverage.test.js

```diff
@@ -0,0 +1,392 @@
+        'type': {
+        'id': '2c91809c7bbdeda7017bbdf524e00022',
+        'value': 'monograph',
+        'label': 'Monograph'
+        },
+        'suppressFromDiscovery': false,
+        'work': {
+        'id': 'a9c4abc5-3744-47ab-9614-27ca3ba54852'
+        },
+        'class': 'org.olf.kb.TitleInstance',
+        'longName': "\"Institutions, industrial upgrading, and economic performance in Japan: the 'flying-geese' paradigm of catch-up growth\"",
+        'relatedTitles': '[]'
+    },
+    'url': 'https://doi.org/10.4337/9781845425678',
+    'name': "'\"Institutions, industrial upgrading, and economic performance in Japan: the 'flying-geese' paradigm of catc...' on Platform 'Elgaronline'",
+    'suppressFromDiscovery': false,
+    'class': 'org.olf.kb.PlatformTitleInstance',
+    'longName': "'\"Institutions, industrial upgrading, and economic performance in Japan: the 'flying-geese' paradigm of catch-up growth\"' on Platform 'Elgaronline'"
+    },
+    'pkg': {
+    'id': '31bd52f9-2ed9-4810-9e59-7e2f5e6638ce',
+    'dateCreated': '2021-09-07T02:04:25Z',
+    'lastUpdated': '2021-09-07T02:04:25Z',
+    'vendor': {
+        'id': '9b21bfe5-d4c0-4025-a76b-78efe16c2505',
+        'name': 'Edward Elgar',
+        'orgsUuid_object': {
+        'error': 400,
+        'message': 'Bad Request'
+        }
+    },
+    'source': 'GOKb',
+    'remoteKb': {
+        'id': 'c683c1f8-ea73-4718-abaf-d26612c98c79',
+        'cursor': '2021-09-06T06:00:42Z',
+        'active': true,
+        'trustedSourceTI': false,
+        'activationEnabled': false,
+        'readonly': false,
+        'syncStatus': 'idle',
+        'lastCheck': 1631000743112,
+        'name': 'GOKb_TEST',
+        'type': 'org.olf.kb.adapters.GOKbOAIAdapter',
+        'fullPrefix': 'gokb',
+        'uri': 'https://gokbt.gbv.de/gokb/oai/index',
+        'supportsHarvesting': true,
+        'rectype': 1
+    },
+    'name': 'Edward Elgar:Edward Elgar E-Book Archive in Business & Management, Economics and Finance:Nationallizenz',
+    'suppressFromDiscovery': false,
+    'reference': 'Edward_Elgar:Edward_Elgar_E-Book_Archive_in_Business_&_Management,_Economics_and_Finance:Nationalliz',
+    'resourceCount': 2540,
+    'class': 'org.olf.kb.Pkg'
+    },
+    'addedTimestamp': 1630980265534,
+    'accessEnd': '2021-03-31',
+    'name': "'\"Institutions, industrial upgrading, and economic performance in Ja...' on Platform 'Elgaronline' in Package Edward Elgar:Edward Elgar E-Book Archive in Business & Management, ...",
+    'lastSeenTimestamp': 1630980265534,
+    'suppressFromDiscovery': false,
+    'longName': "'\"Institutions, industrial upgrading, and economic performance in Japan: the 'flying-geese' paradigm of catch-up growth\"' on Platform 'Elgaronline' in Package Edward Elgar:Edward Elgar E-Book Archive in Business & Management, Economics and Finance:Nationallizenz",
+    'class': 'org.olf.kb.PackageContentItem'
+};
```

Comment:
Lets move this static data into a `testResources` file and import it frmo there.